### DISCUSSION TITLE
Nuget package .NET 6

### DISCUSSION THREAD

**Discussion by TheCraftyMaker** [Blazored/LocalStorage#170] [General]
Hello,

Are there any plans to publish a new nuget package with the .NET 6 compatible sources?

Is there anyway a contributer can takes this up?

Kind regards
Christof

**Comment by chrissainty** [comment]
Apologies for the delay @krikke-21. I've just published the .NET 6 version.

**Reply by TheCraftyMaker** [reply]
Thx @chrissainty. Have a nice weekend.
